recipe/meta.yaml

```diff
@@ -79,6 +79,7 @@ requirements:
     - gtest
     - benchmark
     - entt
+    - sparrow
```

Comment:
This has a pin in the ArcticDB repo's dual of this file https://github.com/man-group/ArcticDB/blob/master/environment-dev.yml#L38 . We should keep them consistent - I think by just dropping the pin in the ArcticDB repo?

Reply:
Yes, for some reason the pin was required for the last version of sparrow to be picked. It is not required anymore. See https://github.com/man-group/ArcticDB/pull/2447.